### PR TITLE
[Doc] add a universal CREATE TABLE sample for window function examples

### PR DESCRIPTION
**Pull request by evelynzhaojie**
Signed-off-by: evelynzhaojie <everlyn.zhaojie@gmail.com

## Why I'm doing:

some examples do not have sample CREATE TABLE and INSERT data.

## What I'm doing:

add CREATE TABLE and INSERT sample. It can be used to test window functions in doc.

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Comment by evelynzhaojie** [comment]
@mergifyio rebase

**Comment by evelynzhaojie** [comment]
@mergifyio rebase

File: docs/en/sql-reference/sql-functions/Window_function.md [hunk 1]
```diff
@@ -134,304 +165,256 @@ select stock_symbol, closing_date, closing_price,
...
+This function is supported from v3.2.
+
+**Syntax:**
 
 ```SQL
 CUME_DIST() OVER (partition_by_clause order_by_clause)
 ```
 
-**This function should be used with ORDER BY to sort partition rows into the desired order. Without ORDER BY, all rows are peers and have value N/N = 1, where N is the partition size.**
+**This function must be used with ORDER BY to sort partition rows into the desired order. Without ORDER BY, all rows are peers and have value `N/N = 1`, where `N` is the partition size.**
```
Comment (EsoragotoSpirit) [inline docs/en/sql-reference/sql-functions/Window_function.md:232]: Without ORDER BY, all rows are peers and have value `N/N = 1`, where `N` is the partition size.

Have the value 

Also, I am not sure what does this is supposed to mean. I didn't find the similar description in CN doc

File: docs/zh/sql-reference/sql-functions/Window_function.md [hunk 2]
```diff
@@ -138,181 +174,285 @@ from stock_ticker;
...
+|    1 | lily  | physics |    60 |  0.3333333333333333 |
+|    5 | mike  | physics |    85 |                 0.5 |
+|    4 | amy   | physics |    99 |  0.8333333333333334 |
+|    3 | jack  | physics |    99 |  0.8333333333333334 |
+|    6 | amber | physics |   100 |                   1 |
++------+-------+---------+-------+---------------------+
+```
+
+- 对于第一行 `cume_dist` 数据 `1`，分组 NULL 中只有这一行数据，小于等于 90 分的数据只有这一行，累积分布为 1。
+- 对于第二行数据 `0.2`，分组 `english` 中有 5 个值，小于等于 NULL 分的数据只有一行 (NULL)，累积分布为 0.2。
```
Comment (EsoragotoSpirit) [inline docs/zh/sql-reference/sql-functions/Window_function.md:277]: 取值为 NULL 的数据只有一行

File: docs/en/sql-reference/sql-functions/Window_function.md [hunk 3]
```diff
@@ -134,304 +165,256 @@ select stock_symbol, closing_date, closing_price,
...
-Example:
-
-We have the following data:
+`IGNORE NULLS` is supported from v2.5.0. It is used to determine whether NULL values of `expr` are eliminated from the calculation. By default, NULL values are included, which means NULL is returned if the last value in the filtered result is NULL. If you specify IGNORE NULLS, the last non-null value in the filtered result is returned. If all the values are NULL, NULL is returned even if you specify IGNORE NULLS.
 
-```SQL
- select name, country, greeting
- from mail_merge;
- ```
+By default, last_value() calculates `rows between unbounded preceding and current row`, which compares the current row with all its preceding rows. If you want to show only one value for each partition, use `rows between unbounded preceding and unbounded following` after ORDER BY.
```
Comment (EsoragotoSpirit) [inline docs/en/sql-reference/sql-functions/Window_function.md:378]: LAST_VALUE()

Maybe we can use consistent letter cases for functions in the doc

File: docs/zh/sql-reference/sql-functions/Window_function.md [hunk 4]
```diff
@@ -389,68 +529,27 @@ FROM test_tbl ORDER BY col_1;
...
-* `default`: 没有找到符合条件的行时，返回的默认值。如果未指定 `default`，默认返回 NULL。`default` 的数据类型必须和 `expr` 兼容。
-* `IGNORE NULLS`：从 3.0 版本开始，`LEAD()` 支持 `IGNORE NULLS`，即是否在计算结果中忽略 NULL 值。如果不指定 `IGNORE NULLS`，默认返回结果会包含 NULL 值。比如，如果指定的当前行之后的第 `offset` 行的值为 NULL，则返回 NULL，参考示例一。如果指定了 `IGNORE NULLS`，向后遍历 `offset` 行时会忽略取值为 NULL 的行，继续向后遍历非 NULL 值。如果指定了 IGNORE NULLS，但是在当前行之后并不存在 offset 个非 NULL 值，则返回 NULL 或 `default` (如果指定)，参考示例二。
+- `expr`: 需要计算的目标字段。
+- `offset`: 偏移量，表示向后查找的行数，必须为**正整数**。如果未指定，默认按照 1 处理。
+- `default`: 没有找到符合条件的行时，返回的默认值。如果未指定 `default`，默认返回 NULL。`default` 的数据类型必须和 `expr` 兼容。
+- `IGNORE NULLS`：从 3.0 版本开始，`LEAD()` 支持 `IGNORE NULLS`，即是否在计算结果中忽略 NULL 值。如果不指定 `IGNORE NULLS`，默认返回结果会包含 NULL 值。比如，如果指定的当前行之后的第 `offset` 行的值为 NULL，则返回 NULL，参考示例一。如果指定了 `IGNORE NULLS`，向后遍历 `offset` 行时会忽略取值为 NULL 的行，继续向后遍历非 NULL 值。如果指定了 IGNORE NULLS，但是在当前行之后并不存在 offset 个非 NULL 值，则返回 NULL 或 `default` (如果指定)，参考示例二。
+
+**示例**：
 
 示例一：lead 中未指定 IGNORE NULLS
```
Comment (EsoragotoSpirit) [inline docs/zh/sql-reference/sql-functions/Window_function.md:554]: LEAD()

File: docs/zh/sql-reference/sql-functions/Window_function.md (hunk 2 above)
Comment (evelynzhaojie) [reply docs/zh/sql-reference/sql-functions/Window_function.md:277]: 上面有描述，”如果数据按升序排列，则统计小于等于当前值的数据在分区的占比“。-- 这里的描述都是小于等于当前值

Comment (EsoragotoSpirit) [reply docs/zh/sql-reference/sql-functions/Window_function.md:277]: 这里我是觉得对于 NULL 这个值没有大于小于等于这个概念，所以描述可以改成“取值为 NULL 的数据只有一行”。不过就是个建议，不重要

Comment (evelynzhaojie) [reply docs/zh/sql-reference/sql-functions/Window_function.md]: 嗯，刚才也考虑到 null 一般都作为最小值处理，这样写会不会不合适，我更新一下。

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Pull request merged**